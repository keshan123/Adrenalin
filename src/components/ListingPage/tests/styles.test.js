import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {
  PageContainer,
  PreviewTile,
  PreviewTileContainer,
  TileImage,
  TileDescription,
  ViewCaseStudyLink,
  ImageTag,
} from '../styles';

test('PageContainer renders', () => {
  const tree = renderer.create(
      <PageContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('PreviewTile renders', () => {
  const tree = renderer.create(
      <PreviewTile/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('PreviewTileContainer renders', () => {
  const tree = renderer.create(
      <PreviewTileContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('TileImage renders', () => {
  const tree = renderer.create(
      <TileImage/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('TileDescription renders', () => {
  const tree = renderer.create(
      <TileDescription/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('ViewCaseStudyLink renders', () => {
  const tree = renderer.create(
      <ViewCaseStudyLink/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('ImageTag renders', () => {
  const tree = renderer.create(
      <ImageTag/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
