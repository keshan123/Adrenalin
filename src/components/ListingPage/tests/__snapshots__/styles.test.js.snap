// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageTag renders 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  background-color: #F5F5F5;
  box-shadow: 0px 4px 4px -2px rgba(0,0,0,0.3);
  color: rgba(0,0,0,0.75);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 12px;
  height: 24px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  left: 0;
  margin-left: -37px;
  margin-top: 37px;
  position: absolute;
  top: 0;
  -webkit-transform: rotate(270deg);
  -ms-transform: rotate(270deg);
  transform: rotate(270deg);
  width: 98px;
}

<div
  className="c0"
/>
`;

exports[`PageContainer renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;

exports[`PreviewTile renders 1`] = `
.c0 {
  color: #ccc;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 517px;
  position: relative;
  width: calc(50% - 16.5px);
}

.c0:nth-child(even) {
  margin-left: 33px;
}

.c0:nth-child(-n+2) {
  margin-bottom: 36px;
}

.c0:nth-last-of-type(-n+2) {
  margin-top: 36px;
}

@media (max-width:768px) {
  .c0 {
    width: 100%;
  }

  .c0:nth-child(even) {
    margin-left: 0px;
    margin-bottom: 36px;
  }

  .c0:nth-child(odd) {
    margin-left: 0px;
    margin-bottom: 36px;
  }

  .c0:nth-last-of-type(-n+2) {
    margin-top: 0px;
  }
}

<div
  className="c0"
/>
`;

exports[`PreviewTileContainer renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  height: 100%;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  left: 0;
  margin-top: 63px;
  padding: 0 93px;
  top: 0;
}

@media (max-width:768px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    padding: 0 45px;
  }
}

<div
  className="c0"
/>
`;

exports[`TileDescription renders 1`] = `
.c0 {
  color: #1A1A1A;
  font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
  font-size: 24px;
  font-weight: bold;
  height: 64px;
  margin-top: 24px;
  text-align: left;
  width: 80%;
}

@media (max-width:1024px) {
  .c0 {
    height: 114px;
  }
}

@media (max-width:768px) {
  .c0 {
    height: 64px;
  }
}

<div
  className="c0"
/>
`;

exports[`TileImage renders 1`] = `
.c0 {
  background-image: url();
  background-size: cover;
  height: 384px;
}

<div
  className="c0"
/>
`;

exports[`ViewCaseStudyLink renders 1`] = `
.c0 {
  color: blue;
  font-size: 12px;
  height: 13px;
  margin-top: 32px;
}

<div
  className="c0"
/>
`;
